forge-gui-desktop/src/main/java/forge/GuiDesktop.java

```diff
@@ -144,15 +144,21 @@ public ISkinImage getCardArt(final PaperCard card, final boolean backFace) {
     }
 
     @Override
-    public ISkinImage createLayeredImage(final FSkinProp background, final String overlayFilename, final float opacity) {
+    public ISkinImage createLayeredImage(final PaperCard paperCard, final FSkinProp background, final String overlayFilename, final float opacity) {
         final BufferedImage image = new BufferedImage(background.getWidth(), background.getHeight(), BufferedImage.TYPE_INT_ARGB);
         final Graphics2D g = image.createGraphics();
         final FSkin.SkinImage backgroundImage = FSkin.getImage(background);
         FSkin.drawImage(g, backgroundImage, 0, 0, background.getWidth(), background.getHeight());
+        final int cardImageWidth = 90;
+        final int cardImageHeight = 128;
 
         if (FileUtil.doesFileExist(overlayFilename)) {
             final ImageIcon overlay = new ImageIcon(overlayFilename);
             g.drawImage(overlay.getImage(), (background.getWidth() - overlay.getIconWidth()) / 2, (background.getHeight() - overlay.getIconHeight()) / 2, overlay.getIconWidth(), overlay.getIconHeight(), null);
+        } else {
```

Comment:
seems it still needs NPE protection here

```
EDT > java.lang.NullPointerException: Cannot invoke "forge.item.PaperCard.getCardImageKey()" because "paperCard" is null
	at forge.GuiDesktop.createLayeredImage(GuiDesktop.java:159)
	at forge.localinstance.achievements.Achievement.updateTrophyImage(Achievement.java:158)
	at forge.localinstance.achievements.Achievement.getImage(Achievement.java:78)
	at forge.screens.home.settings.VSubmenuAchievements$TrophyCase.paintComponent(VSubmenuAchievements.java:341)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1074)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JViewport.paint(JViewport.java:737)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at forge.toolbox.FScrollPane.paint(FScrollPane.java:96)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
```

Reply:
done